addons/hr_holidays/models/calendar_event.py

```diff
@@ -0,0 +1,12 @@
+from odoo import models
+
+
+class CalendarEvent(models.Model):
+    _inherit = 'calendar.event'
+
+    def _need_video_call(self):
+        """ Determine if the event needs a video call or not depending
+        on the model of the event.
+        """
+        self.ensure_one()
+        return self.res_model != 'hr.leave'
```

Comment:
You should not force the behavior of other module. super must be called to have a proper override :-).
```suggestion
    def _need_video_call(self):
        """ Determine if the event needs a video call or not depending
        on the model of the event.
        """
        self.ensure_one()
        if  self.res_model == 'hr.leave':
             return False
        return  super()._need_video_call()
```